src/components/Loader/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,64 @@
+import { useEffect, useState } from 'react';
+import { Loader } from '../Loader';
+import { PeopleList } from './PeopleList';
+import { getPeople } from '../../../api';
+import { Person } from '../../../types';
+
+export const PeopleTable = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
+
+  useEffect(() => {
+    setIsLoading(true);
+
+    getPeople()
+      .then(setPeople)
+      .catch(() => {
+        setIsError(true);
+      })
+      .finally(() => {
+        setIsLoading(false);
+      });
+
+    return () => {
+      setPeople([]);
+    };
+  }, []);
+
+  const preparedPeople = people.map(person => {
+    const newPerson = { ...person };
+
+    const mother = people.find(m => person.motherName === m.name);
+    const father = people.find(f => person.fatherName === f.name);
+
+    newPerson.mother = mother;
+    newPerson.father = father;
+
+    return newPerson;
```

Comment:
The `map` operation here will run on every render since it's not memoized. If the computation of `preparedPeople` is expensive, you could use `useMemo` to only recompute when `people` changes.